packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/configlocations.md

```diff
@@ -0,0 +1,132 @@
+There are several options to set the sample app's configuration.
+
+### Option 1: YAML configuration
+
+Create a YAML file named `okta.yaml` in one of the following three directories:
+
+* Current user's home directory.
+  * **Unix/Linux**:    `~/.okta/okta.yaml`
+  * **Windows**:       `%userprofile%\.okta\okta.yaml`
+
+* Application or project's root directory
+
+The following is the required content format for the YAML file:
+
+```yaml
+okta:
+  idx:
+    issuer: "https://{yourOktaDomain}/oauth2/{authorizationServerId}"
+    clientId: "{clientId}"
+    clientSecret: "{clientSecret}"
+    scopes:
+    - "{scope1}"
+    - "{scope2}"
+    redirectUri: "{redirectUri}"
+```
+
+YAML configuration file example:
+
+```yaml
+okta:
+  idx:
+    issuer: "https://dev-1234.okta.com/oauth2/default"
+    clientId: "123xyz"
+    clientSecret: "123456abcxyz"
+    scopes:
+    - "openid"
+    - "profile"
+    - "offline_access"
+    redirectUri: "http://localhost:8080"
+```
+
+Where
+
+* `{authorizationServerId}=default` (the default Custom Authorization Server)
+* `{yourOktaDomain}=dev-1234.okta.com`
+* `{clientId}=123xyz`
+* `{clientSecret}=123456abcxyz`
+* `{redirectUri}=http://localhost:8080`
+* `openid`, `profile`, and `offline_access` scopes are used for the sample app
+
+### Option 2: Environment variables
+
+#### Embedded authentication with SDK sample app
+
+Set the following environment variables with your app's configuration values before running the embedded authentication with SDK sample app:
+
+* `OKTA_IDX_ISSUER`
+* `OKTA_IDX_CLIENTID`
+* `OKTA_IDX_CLIENTSECRET`
+* `OKTA_IDX_REDIRECTURI`
+* `OKTA_IDX_SCOPES`
+
+The following is an example of setting environment variables for the embedded authentication with SDK sample app, in bash or zsh:
```

Comment:
Nit: Change `in bash or zsh` -> `in shell`.

Reply:
Fixed in new branch vn-OKTA-408182-oie-java-embedded-app-2.